perception/autoware_detection_by_tracker/src/tracker/tracker_handler.hpp

```diff
@@ -32,7 +32,7 @@ class TrackerHandler
 public:
   TrackerHandler() = default;
   void onTrackedObjects(
-    const autoware_perception_msgs::msg::TrackedObjects::ConstSharedPtr input_objects_msg);
+    const autoware_perception_msgs::msg::TrackedObjects::ConstSharedPtr msg);
```

Comment:
Is it possible to align it with input_objects_msg? The name msg is too generic.

Reply:
Thanks for the advice, I have modified it to match input_objects_msg.